server/src/clang-utils/SourceToHeaderMatchCallback.cpp

```diff
@@ -209,6 +210,17 @@ void SourceToHeaderMatchCallback::generateInternal(const FunctionDecl *decl) con
         wrapperDecl = getOldStyleDeclarationAsString(decl, wrapperName);
     }
 
+    auto returnType = decl->getReturnType();
+    if (returnType->hasUnnamedOrLocalType()) {
+        if (auto EN = llvm::dyn_cast<clang::EnumDecl>(returnType->getAsTagDecl())) {
```

Comment:
Change name EN on more understandable (for example enumStmt)
Also you can declare and define this variable before first if statement and use it in condition and in if body.

Reply:
Done